dreamboat/main.py

```diff
@@ -0,0 +1,90 @@
+import pycurl
+import json
+import time
+import logging
+import ConfigParser
+import thread
+
+CONFIG_FILE = 'config'
+
+def make_logger(log_name, log_level):
+    # setup logger
+    logger = logging.getLogger(log_name)
+    logger.setLevel(log_level)
+    # create console handler and set level to debug
+    ch = logging.StreamHandler()
+    ch.setLevel(logging.DEBUG)
+    # create formatter
+    formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+    # add formatter to ch
+    ch.setFormatter(formatter)
+    # add ch to logger
+    logger.addHandler(ch)
+    return logger
+
+def give_life(source_url, data, DELAY, log_level):
+    log_name = 'heartbeat_{0}'.format(source_url)
+    logger = make_logger(log_name, log_level)
+    while True:
+        try:
+            c = pycurl.Curl()
+            c.setopt(pycurl.URL, source_url)
+            c.setopt(pycurl.HTTPHEADER, ['Content-Type: application/json'])
```

Comment:
I had suggested we add a header here (X-22acaia-heartbeat or something) as we need something to disambiguate these heartbeats from valid messages.  

I do see that the data is being defaulted to '_hearbeat_: yes' but I would prefer we give ourselves the option of excluding this sample data not in the processing or business logic but in the system layer.  this way we never have to write exclusion code in the business logic.  because otherwise we'll have to start every bit of business logic 'if data.is_heartbeat? eject into the sun'  And I'd rather we not force that boilerplate onto ourselves.

if we want to have the boilerplate for now, fine, there's only four of us and the product we're building is barely in the second trimester so future code clarity isn't required but enabling future code clarity should be.